.github/workflows/update_image.yml

```diff
@@ -20,7 +19,7 @@ jobs:
     env:
       AW_DOCKER_USERNAME: ${{ secrets.AW_Docker_Username }}
```

Comment:
Flagging that we should change this to a general NYCPlanning docker username. Currently, we are using Sasha's username/password and when we push to DockerHub Sasha is still the author [see here](https://registry.hub.docker.com/r/nycplanning/library/tags). Not an immediate change that needs to happen though